src/types/entity/other.ts

```diff
@@ -160,3 +160,52 @@ export interface RecommendPost {
   createdAt: Date
   type: string
 }
+
+export interface FollowingFeed {
+  type: string
```

Comment:
You can use `UnionLiteral` to list the possible options whenever possible.
```suggestion
  type: UnionLiteral<'maybe-type'> 
```

Reply:
Also, please put `FollowingFeed` to `src/types/api-responses.ts`. Just make it merged to `FollowingFeedResponse`

It seems unlikely to be used twice.